### ISSUE TITLE
Updating port attrs

### ISSUE THREAD

**Issue by matthewatabet** [clientIO/joint#360]
Is it possible to update the attrs of a port after a port has been created? I'm using the new port api defined in `port.js`.

I tried something like this:

```
var port = element.getPort('in_1')
port.attrs['.port-body']['fill-opacity'] = 1;
element.trigger('change:ports');
```

But, that didn't seem to trigger a re-render of the ports. Thanks!

**Comment by matthewatabet** [comment]
The solution here was to find the port within the element's nested props:

```
element.prop('ports/items/2/attrs/.port-body/fill-opacity', 1)
```

Works like a charm. Thanks!

**Comment by kumilingus** [comment]
Hi Matthew,

I think you have pointed at a deficiency here. In order to change a specific port attributes, one has to get the index of the port first and then build the `path` and set the value.

We should probably introduce another API method that would help with these cases.

For instance:

```javascript
    element.portProp('port-id', 'attrs/.port-body/fill-opacity', 1);
    // alternatively
    element.portProp('port-id', 'attrs/.port-body', { 'fill-opacity':  1 });
````

Thank you for the feedback.

**Comment by kumilingus** [comment]
Fixed in 75914231b41b8c8bb23000041dde3644447a45ce.
